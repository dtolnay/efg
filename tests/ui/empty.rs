use efg::efg;

#[efg()]
struct S;

fn main() {}
