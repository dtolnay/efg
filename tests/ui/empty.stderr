error: unexpected end of input
 --> tests/ui/empty.rs:3:1
  |
3 | #[efg()]
  | ^^^^^^^^
  |
  = note: this error originates in the attribute macro `efg` (in Nightly builds, run with -Z macro-backtrace for more info)
